src/main/groovy/com/github/livepvrdata/MonitorFactory.groovy

```diff
@@ -48,6 +48,12 @@ class MonitorFactory {
 			case ~/NFL Football/:
 				monitor = new NFL(req.details, 1000L * req.start)
 				break
+			case ~/Formula One Racing|Formula One/:
```

Comment:
You don't really need both of these conditions, "Formula One" covers both of them.